### DISCUSSION TITLE
How change screen output color

### DISCUSSION THREAD

**Discussion by Mahmoudgaberbadr** [cssbattle/forum#266] [Q&A]
How change screen output color from white  to make background for shape???

**Comment by gunnarbittersmann** [comment]
Set `html{background:#F3AC3C}` or `body{background:#F3AC3C}`. On many targets you can set it for all elements: `*{background:#F3AC3C}`. On some targets it pays off to use `<body bgcolor=F3AC3C>` from pre-CSS days.
